kie-remote/src/main/java/org/kie/remote/impl/producer/RemoteCepKieSessionImpl.java

```diff
@@ -23,13 +23,11 @@
 import org.kie.remote.RemoteCepKieSession;
 import org.kie.remote.TopicsConfig;
 
-import static org.kie.remote.impl.producer.RemoteKieSessionImpl.DEFAULT_ENTRY_POINT;
```

Comment:
Same question as above

Reply:
No, it is not probably some automatic IDE change during refactoring. Do you see it as a problem?

Reply:
Same answer :smile: